app/settings.php

```diff
@@ -26,7 +26,6 @@
                     'subjectParams' => ['donorFirstName'],
                     'requiredParams' => [
                         'campaignName',
-                        'campaignThankYouMessage',
```

Comment:
no need to change the template for this email as there is already an if condition around this field, even though it's marked as required here: https://github.com/thebiggive/mailer/blob/d1cee238733e5d5176bbd52b79c8b8373fd80568/templates/donor-donation-success.html.twig#L8